aea/crypto/ledger_apis.py

```diff
@@ -25,12 +25,17 @@
 
 from fetchai.ledger.api import LedgerApi as FetchLedgerApi  # type: ignore
 # from fetchai.ledger.api.tx import TxStatus
-from fetchai.ledger.crypto import Entity, Identity, Address  # type: ignore
+from fetchai.ledger.crypto import Identity, Address  # type: ignore
+from web3 import Web3, HTTPProvider  # type: ignore
 
+from aea.crypto.base import Crypto
 from aea.crypto.ethereum import ETHEREUM
 from aea.crypto.fetchai import FETCHAI
 
 DEFAULT_FETCHAI_CONFIG = ('alpha.fetch-ai.com', 80)
+ETHEREUM_TEST_NETWORK = ("https://ropsten.infura.io/v3/f00f7b3ba0e848ddbdc8941c527447fe",)
```

Comment:
do we really want to commit this to the AEA package? Put differently, is this expected to work for the foreseeable future?